src/server/api/question.js

```diff
@@ -25,6 +25,7 @@ export const resolvers = {
       r.table('interaction_step')
         .filter({ parent_interaction_id: interactionStep.id })
         .filter({ is_deleted: false })
+        .orderBy('answer_option')
```

Comment:
what if this was `.orderBy('name', 'desc')` in order to make sure 'yes' or yes answers are first.